src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/locations/LocationsReconciliationService.kt

```diff
@@ -277,10 +279,16 @@ class LocationsReconciliationService(
       return true
     }
     val nomisListConverted = filteredAmendmentResponses?.map { a ->
-      ChangeHistory(toHistoryAttribute(a.columnName), a.amendedBy, a.amendDateTime, a.oldValue, a.newValue)
+      ChangeHistory(
+        toHistoryAttribute(a.columnName),
+        a.amendedBy,
+        a.amendDateTime,
+        UUID.fromString(a.oldValue),
+        a.newValue?.let { ChangeHistory.TransactionType.valueOf(a.newValue!!) },
+      )
     } ?: emptyList()
-    val nomisSetSorted = nomisListConverted.sortedBy { it.amendedDate + it.attribute + it.amendedBy + it.oldValue + it.newValue }
-    val dpsSetSorted = filteredDpsHistoryResponses?.sortedBy { it.amendedDate + it.attribute + it.amendedBy + it.oldValue + it.newValue } ?: emptyList()
+    val nomisSetSorted = nomisListConverted.sortedBy { it.amendedDate.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME) + it.attribute + it.amendedBy }
```

Comment:
This was weird.
We were grabbing `it.oldValue + it.newValue` but we hadn't set those values in the constructor above.
Also DPS has changed them to `oldValues` and `newValues` so they would never have been compared anyway